### PR TITLE
[zh-cn] sync blog/2022-08-31-cgroupv2-ga.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn] sync blog/2022-08-31-cgroupv2-ga.md

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+**Authors:**: David Porter (Google), Mrunal Patel (Red Hat)
+-->
+**作者**: David Porter (Google), Mrunal Patel (Red Hat)
+
+<!--
+Kubernetes 1.25 brings cgroup v2 to GA (general availability), letting the
+[kubelet](/docs/concepts/overview/components/#kubelet) use the latest container resource
+management capabilities.
+-->
+Kubernetes 1.25 将 cgroup v2 升级为一般可用性（GA），
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:25]:
```suggestion
Kubernetes 1.25 将 cgroup v2 升级为正式发布（GA），
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+your nodes, such as CPU, memory, and storage.
+
+*cgroups* are a Linux kernel capability that establish resource management
+functionality like limiting CPU usage or setting memory limits for running
+processes.
+-->
+有效的[资源管理](/zh-cn/docs/concepts/configuration/manage-resources-containers/)是 Kubernetes 的一个关键方面。
+这涉及管理节点中的有限资源，例如 CPU、内存和存储。
+
+*cgroups* 是一种 Linux 内核功能，可建立资源管理功能，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:45]:
```suggestion
**cgroups** 是一种 Linux 内核功能，可建立资源管理功能，
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+cgroup v2 has been in development in the Linux Kernel since 2016 and in recent
+years has matured across the container ecosystem. With Kubernetes 1.25, cgroup
+v2 support has graduated to general availability.
+-->
+cgroup v2 是 Linux cgroup API 的最新版本,
+提供了一个具有增强的资源管理能力的统一控制系统。
+
+自 2016 年以来，cgroup v2 一直在 Linux 内核中进行开发，
+近年来在整个容器生态系统中已经成熟。 在 Kubernetes 1.25 中，
+对 cgroup v2 的支持已升级为一般可用。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:79]:
```suggestion
对 cgroup v2 的支持已升级为正式发布。
```
Refer to [glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7)

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+* Single unified hierarchy design in API
+* Safer sub-tree delegation to containers
+* Newer features like [Pressure Stall Information](https://www.kernel.org/doc/html/latest/accounting/psi.html)
+* Enhanced resource allocation management and isolation across multiple resources
+    * Unified accounting for different types of memory allocations (network and kernel memory, etc)
+    * Accounting for non-immediate resource changes such as page cache write backs
+-->
+* API 中的单一统一层次结构设计
+* 更安全的子树委派给容器
+* [压力失速信息](https://www.kernel.org/doc/html/latest/accounting/psi.html)等新功能
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:103]:
```suggestion
* [压力阻塞信息](https://www.kernel.org/doc/html/latest/accounting/psi.html)等新功能
```
Refer to [cgroups](https://kubernetes.io/zh-cn/docs/concepts/architecture/cgroups/#cgroup-v2) for this para.

**Comment by windsonsea** [comment]
Refer to [glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7) and [cgroups](https://kubernetes.io/zh-cn/docs/concepts/architecture/cgroups/#cgroup-v2).

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+
+推荐使用默认使用 cgroup v2 的 Linux 发行版。 
+一些使用 cgroup v2 的流行 Linux 发行版包括：
+
+* Container Optimized OS (since M97)
+* Ubuntu (since 21.10)
+* Debian GNU/Linux (since Debian 11 Bullseye)
+* Fedora (since 31)
+* Arch Linux (since April 2021)
+* RHEL and RHEL-like distributions (since 9)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:151]: Need localize

**Review by windsonsea — COMMENTED** [review]
Did you really read https://kubernetes.io/zh-cn/docs/concepts/architecture/cgroups/#cgroup-v2 ?
Much content is same there.

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+<!--
+* Single unified hierarchy design in API
+* Safer sub-tree delegation to containers
+* Newer features like [Pressure Stall Information](https://www.kernel.org/doc/html/latest/accounting/psi.html)
+* Enhanced resource allocation management and isolation across multiple resources
+    * Unified accounting for different types of memory allocations (network and kernel memory, etc)
+    * Accounting for non-immediate resource changes such as page cache write backs
+-->
+* API 中的单一统一层次结构设计
+* 更安全的子树委派给容器
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:102]:
```suggestion
* 为容器提供更安全的子树委派能力
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+<!--
+## Learn more
+
+* Read the [Kubernetes cgroup v2 documentation](/docs/concepts/architecture/cgroups/)
+* Read the enhancement proposal, [KEP 2254](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2254-cgroup-v2/README.md)
+* Learn more about
+  [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) on Linux Manual Pages
+  and [cgroup v2](https://docs.kernel.org/admin-guide/cgroup-v2.html) on the Linux Kernel documentation
+-->
+## 学到更多
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:252]:
```suggestion
## 进一步了解
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+your nodes, such as CPU, memory, and storage.
+
+*cgroups* are a Linux kernel capability that establish resource management
+functionality like limiting CPU usage or setting memory limits for running
+processes.
+-->
+有效的[资源管理](/zh-cn/docs/concepts/configuration/manage-resources-containers/)是 Kubernetes 的一个关键方面。
+这涉及管理节点中的有限资源，例如 CPU、内存和存储。
+
+**cgroups** 是一种 Linux 内核功能，可建立资源管理功能，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:45]:
```suggestion
**cgroups** 是一种可建立资源管理功能的 Linux 内核能力，
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+例如限制 CPU 使用率或为正在运行的进程设置内存限制。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:46]:
```suggestion
例如为正在运行的进程限制 CPU 使用率或设置内存限制。
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 10]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+When you use the resource management capabilities in Kubernetes, such as configuring
+[requests and limits for Pods and containers](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits),
+Kubernetes uses cgroups to enforce your resource requests and limits.
+
+The Linux kernel offers two versions of cgroups: cgroup v1 and cgroup v2.
+-->
+当你使用 Kubernetes 中的资源管理功能时，例如配置
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:55]:
```suggestion
当你使用 Kubernetes 中的资源管理能力时，例如配置 
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 11]
```diff
@@ -0,0 +1,305 @@
...
+
+cgroup v2 has been in development in the Linux Kernel since 2016 and in recent
+years has matured across the container ecosystem. With Kubernetes 1.25, cgroup
+v2 support has graduated to general availability.
+-->
+cgroup v2 是 Linux cgroup API 的最新版本,
+提供了一个具有增强的资源管理能力的统一控制系统。
+
+自 2016 年以来，cgroup v2 一直在 Linux 内核中进行开发，
+近年来在整个容器生态系统中已经成熟。 在 Kubernetes 1.25 中，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:78]:
```suggestion
近年来在整个容器生态系统中已经成熟。在 Kubernetes 1.25 中，
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+* Single unified hierarchy design in API
+* Safer sub-tree delegation to containers
+* Newer features like [Pressure Stall Information](https://www.kernel.org/doc/html/latest/accounting/psi.html)
+* Enhanced resource allocation management and isolation across multiple resources
+    * Unified accounting for different types of memory allocations (network and kernel memory, etc)
+    * Accounting for non-immediate resource changes such as page cache write backs
+-->
+* API 中的单一统一层次结构设计
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:101]:
```suggestion
* API 中单个统一的层次结构设计
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 13]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+Some Kubernetes features exclusively use cgroup v2 for enhanced resource
+management and isolation. For example,
+the [MemoryQoS feature](/blog/2021/11/26/qos-memory-resources/) improves
+memory utilization and relies on cgroup v2 functionality to enable it. New
+resource management features in the kubelet will also take advantage of the new
+cgroup v2 features moving forward.
+-->
+一些 Kubernetes 功能专门使用 cgroup v2 来增强资源管理和隔离。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:116]:
```suggestion
一些 Kubernetes 特性专门使用 cgroup v2 来增强资源管理和隔离。 
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 14]
```diff
@@ -0,0 +1,305 @@
...
+<!--
+Some Kubernetes features exclusively use cgroup v2 for enhanced resource
+management and isolation. For example,
+the [MemoryQoS feature](/blog/2021/11/26/qos-memory-resources/) improves
+memory utilization and relies on cgroup v2 functionality to enable it. New
+resource management features in the kubelet will also take advantage of the new
+cgroup v2 features moving forward.
+-->
+一些 Kubernetes 功能专门使用 cgroup v2 来增强资源管理和隔离。 
+例如，[MemoryQoS 功能](/blog/2021/11/26/qos-memory-resources/)提高了内存利用率并依赖
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:117]:
```suggestion
例如，[MemoryQoS 特性](/blog/2021/11/26/qos-memory-resources/)提高了内存利用率并依赖
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 15]
```diff
@@ -0,0 +1,305 @@
...
+cgroup v2 功能来启用它。kubelet 中的新资源管理功能也将利用新的 cgroup v2 功能向前发展。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:118]:
```suggestion
cgroup v2 功能来启用它。kubelet 中的新资源管理特性也将利用新的 cgroup v2 特性向前发展。
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 16]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+* Your Linux distribution enables cgroup v2 on kernel version 5.8 or later
+* Your container runtime supports cgroup v2. For example:
+  * [containerd](https://containerd.io/) v1.4 or later
+  * [cri-o](https://cri-o.io/) v1.20 or later
+* The kubelet and the container runtime are configured to use the [systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver)
+-->
+* 你的 Linux 发行版在内核版本 5.8 或更高版本上启用 cgroup v2
+* 你的容器运行时支持 cgroup v2。 例如：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:179]:
```suggestion
* 你的容器运行时支持 cgroup v2。例如：
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 17]
```diff
@@ -0,0 +1,305 @@
...
+The kubelet and container runtime use a [cgroup driver](/docs/setup/production-environment/container-runtimes#cgroup-drivers)
+to set cgroup paramaters. When using cgroup v2, it's strongly recommended that both
+the kubelet and your container runtime use the
+[systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver),
+so that there's a single cgroup manager on the system. To configure the kubelet
+and the container runtime to use the driver, refer to the
+[systemd cgroup driver documentation](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver).
+-->
+kubelet 和容器运行时使用 [cgroup 驱动](/zh-cn/docs/setup/production-environment/container-runtimes#cgroup-drivers) 
+来设置 cgroup 参数。 使用 cgroup v2 时，强烈建议 kubelet 和你的容器运行时都使用
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:194]:
```suggestion
来设置 cgroup 参数。使用 cgroup v2 时，强烈建议 kubelet 和你的容器运行时都使用 
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 18]
```diff
@@ -0,0 +1,305 @@
...
+the kubelet and your container runtime use the
+[systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver),
+so that there's a single cgroup manager on the system. To configure the kubelet
+and the container runtime to use the driver, refer to the
+[systemd cgroup driver documentation](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver).
+-->
+kubelet 和容器运行时使用 [cgroup 驱动](/zh-cn/docs/setup/production-environment/container-runtimes#cgroup-drivers) 
+来设置 cgroup 参数。 使用 cgroup v2 时，强烈建议 kubelet 和你的容器运行时都使用 
+[systemd cgroup 驱动程序](/zh-cn/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver)，
+以便有一个 cgroup 系统管理员。 要配置 kubelet 和容器运行时以使用驱动程序，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:196]:
```suggestion
以便系统上只有一个 cgroup 管理员。要配置 kubelet 和容器运行时以使用该驱动程序，
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 19]
```diff
@@ -0,0 +1,305 @@
...
+Scenarios in which you might need to update to cgroup v2 include the following:
+
+* If you run third-party monitoring and security agents that depend on the cgroup file system, update the
+  agents to versions that support cgroup v2.
+* If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone
+  DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
+* If you deploy Java applications with the JDK, prefer to use JDK 11.0.16 and
+  later or JDK 15 and later, which [fully support cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305).
+-->
+如果你有直接访问 cgroup 文件系统的应用程序，无论是在节点上还是从容器内部，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:232]:
```suggestion
如果你在节点上或从容器内直接访问 cgroup 文件系统的应用程序，
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 20]
```diff
@@ -0,0 +1,305 @@
...
+你必须更新应用程序以使用 cgroup v2 API 而不是 cgroup v1 API。
+
+你可能需要更新到 cgroup v2 的场景包括：
+
+* 如果你运行依赖于 cgroup 文件系统的第三方监控和安全代理，请将代理更新到支持 cgroup v2 的版本。
+* 如果你将 [cAdvisor](https://github.com/google/cadvisor) 作为独立的 DaemonSet 运行以监控 Pod 和容器，
+  请将其更新到 v0.43.0 或更高版本。
+* 如果你使用 JDK 部署 Java 应用程序，首选使用 JDK 11.0.16 及更高版本或 JDK 15 及更高版本，
+  [完全支持 cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305)。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:241]:
```suggestion
* 如果你使用 JDK 部署 Java 应用程序，首选使用[完全支持 cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305)的
  JDK 11.0.16 及更高版本或 JDK 15 及更高版本。
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 21]
```diff
@@ -0,0 +1,305 @@
...
+* Read the [Kubernetes cgroup v2 documentation](/docs/concepts/architecture/cgroups/)
+* Read the enhancement proposal, [KEP 2254](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2254-cgroup-v2/README.md)
+* Learn more about
+  [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) on Linux Manual Pages
+  and [cgroup v2](https://docs.kernel.org/admin-guide/cgroup-v2.html) on the Linux Kernel documentation
+-->
+## 进一步了解
+
+* 阅读 [Kubernetes cgroup v2 文档](/zh-cn/docs/concepts/architecture/cgroups/)
+* 阅读增强提案，[KEP 2254](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2254-cgroup-v2/README.md)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:255]:
```suggestion
* 阅读增强提案 [KEP 2254](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2254-cgroup-v2/README.md)
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 22]
```diff
@@ -0,0 +1,305 @@
...
+* 学习更多关于 Linux 手册页上的 [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) 和Linux 内核文档上的
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:256]:
```suggestion
* 学习更多关于 Linux 手册页上的 [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) 和 Linux 内核文档上的
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 23]
```diff
@@ -0,0 +1,305 @@
...
+using the SIG [mailing list](https://github.com/kubernetes/community/tree/master/sig-node#contact).
+
+cgroup v2 has had a long journey and is a great example of open source
+community collaboration across the industry because it required work across the
+stack, from the Linux Kernel to systemd to various container runtimes, and (of
+course) Kubernetes.
+-->
+## 参与其中
+
+随时欢迎你的反馈！SIG 节点定期开会，可在 Kubernetes [Slack](https://slack.k8s.io/)的
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:273]:
```suggestion
随时欢迎你的反馈！SIG Node 定期开会，可在 Kubernetes [Slack](https://slack.k8s.io/)的 
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 24]
```diff
@@ -0,0 +1,305 @@
...
+[cAdvisor](https://github.com/google/cadvisor)
+and [runc, libcontainer](https://github.com/opencontainers/runc),
+which underpin many container runtimes. Finally, this wouldn't have been
+possible without support from systemd and upstream Linux Kernel maintainers.
+
+It's a team effort!
+-->
+## 致谢
+
+我们要感谢在 Kubernetes 中发起 cgroup v2 支持的 [Giuseppe Scrivano](https://github.com/giuseppe)，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:298]:
```suggestion
我们要感谢 [Giuseppe Scrivano](https://github.com/giuseppe) 在 Kubernetes 中发起对 cgroup v2 的支持，
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 25]
```diff
@@ -0,0 +1,305 @@
...
+以及来自 SIG Node 社区的引领和审查，包括主席 [Dawn Chen](https://github. com/dchen1107) 和
+[Derek Carr](https://github.com/derekwaynecarr)。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:300]:
```suggestion
还要感谢 SIG Node 社区主席 [Dawn Chen](https://github. com/dchen1107) 和
[Derek Carr](https://github.com/derekwaynecarr) 所作的审查和领导工作。
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 26]
```diff
@@ -0,0 +1,305 @@
...
+* Learn more about
+  [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) on Linux Manual Pages
+  and [cgroup v2](https://docs.kernel.org/admin-guide/cgroup-v2.html) on the Linux Kernel documentation
+-->
+## 进一步了解
+
+* 阅读 [Kubernetes cgroup v2 文档](/zh-cn/docs/concepts/architecture/cgroups/)
+* 阅读增强提案 [KEP 2254](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2254-cgroup-v2/README.md)
+* 学习更多关于 Linux 手册页上的 [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) 和 Linux 内核文档上的
+  [cgroup v2](https://docs.kernel.org/admin-guide/ cgroup-v2.html）
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:257]:
```suggestion
  [cgroup v2](https://docs.kernel.org/admin-guide/cgroup-v2.html）
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 27]
```diff
@@ -0,0 +1,305 @@
...
+
+cgroup v2 has had a long journey and is a great example of open source
+community collaboration across the industry because it required work across the
+stack, from the Linux Kernel to systemd to various container runtimes, and (of
+course) Kubernetes.
+-->
+## 参与其中
+
+随时欢迎你的反馈！SIG Node 定期开会，可在 Kubernetes [Slack](https://slack.k8s.io/)的 
+`#sig-node` 频道中获得，或使用 SIG [邮件列表](https://github. com/kubernetes/community/tree/master/sig-node#contact)。
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:274]:
```suggestion
`#sig-node` 频道中获得，或使用 SIG [邮件列表](https://github.com/kubernetes/community/tree/master/sig-node#contact)。
```

File: content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md [hunk 28]
```diff
@@ -0,0 +1,305 @@
...
+and [runc, libcontainer](https://github.com/opencontainers/runc),
+which underpin many container runtimes. Finally, this wouldn't have been
+possible without support from systemd and upstream Linux Kernel maintainers.
+
+It's a team effort!
+-->
+## 致谢
+
+我们要感谢 [Giuseppe Scrivano](https://github.com/giuseppe) 在 Kubernetes 中发起对 cgroup v2 的支持，
+还要感谢 SIG Node 社区主席 [Dawn Chen](https://github. com/dchen1107) 和
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md:299]:
```suggestion
还要感谢 SIG Node 社区主席 [Dawn Chen](https://github.com/dchen1107) 和
```

**Pull request merged**